LICENSE.md

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2018 Alexander Sagen
+Copyright (c) 2018 Alexander Sagen,
```

Comment:
I don't think there should be a comma here, although I'm not sure if it matters.